services/apps/portal/src/main.ts

```diff
@@ -53,6 +54,19 @@ const registerPipes = (app: NestFastifyApplication) => {
   );
 };
 
+const registerMultiPart = async (app: NestFastifyApplication) => {
+  await app.register(fastifyMultipart, {
+    limits: {
+      fieldNameSize: 100, // Max field name size in bytes
+      fieldSize: 100, // Max field value size in bytes
+      fields: 10, // Max number of non-file fields
+      fileSize: config.get<number>('STORAGE.MAX_FILE_SIZE'), // For multipart forms, the max file size in bytes
+      files: 1, // Max number of file fields
```

Comment:
I thought the idea was to have multiple files upload. Is it changed?

Reply:
There are pretty much 0 requirments, we dont even show the file once it is seleted/uploaded. We might need to change this later but I also imagine we would be uploading documents one at a time when going through the process.